### DISCUSSION TITLE
BETA Testing started!

### DISCUSSION THREAD

**Discussion by NilsSve** [NilsSve/DfRefactor#64] [Announcements]
Beta testing has commenced. This will be a relatively short period and it is expected that the release will be at the beginning of March.

**Comment by SoftAllan** [comment]
Hi Nils.
I've started testing and for now only the "unused source files" refactor function. I have found that it does not handle the UTF-8 BOM very well. I'll have a look at it to see, if I can fix this.

**Reply by SoftAllan** [reply]
Hi @NilsSve. What would be the best way to address this? I would like to contribute to this and I have already forked the project. And I will work on it tomorrow. But should I create an Issue here on this project and then start working on it on my won forked version? Or how do one usually start fixing a problem that has not yet been created and assigned in the original project like here?

**Comment by NilsSve** [comment]
@SoftAllan Yes, I think the best would be if you would report the problem as an "Issue" at GitHub and then start working on your forked version. Then when ready, probably the best would be to send me the changes privately so I can check in the changes? 

For this project, we have not yet had any contributors to make any commits & pushes. Almost all issues have been fixed by me. But I appreciate if you want to fix the issue!

**Reply by SoftAllan** [reply]
Sure no problem. However I have worked quite a bit with Pull Request in Microsoft DevOps. I think it is not much different here in GitHub. If you like we could try that I make a Pull Request from my forked version when I have something to submit? Otherwise I am fine with sending you the code changes.

**Comment by NilsSve** [comment]
Sure, let's try to make a Pull Request. We can always make an undo if something goes wrong.

**Comment by NilsSve** [comment]
The repo is now in "Release" state.
So if someone would like to test it and contribute, it would be highly appreciated!

**Comment by NilsSve** [comment]
The beta testing has been closed as the repo is now in "Release" state
